styles.css

```diff
@@ -551,6 +1216,16 @@
   margin-bottom: 25px;
 }
 
+@media (min-width: 992px) {
+  .collaboration_form_button {
+    position: absolute;
+    width: 254px;
+    height: 70px;
+    left: 593px;
+    top: 650px;
+  }
+}
+
```

Comment:
## WELL DONE ✅😲
- [ ] [OPTIONAL] All the things are fixed from the last review 👏😀. I also have a little suggestion to help improve your design. I recommend changing the cursor to pointer 👆 when hovering buttons. **TIP 📌**: You can add `cursor: pointer:`